onnx/backend/test/case/node/flatten.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Flatten(Base):
+
+    @staticmethod
+    def export():
+        shape = (2, 3, 4, 5)
+        a = np.random.random_sample(shape).astype(np.float32)
+
+        for i in range(len(shape)):
+            node = onnx.helper.make_node(
+                'Flatten',
+                inputs=['a'],
+                outputs=['b'],
+                axis=i,
+            )
+            newshape = (1, -1) if i == 0 else (np.prod(shape[0:i]).astype(int), -1)
+            b= np.reshape(a, newshape)
+            expect(node, inputs=[a], outputs=[b],
+               name='test_flatten_axis' + str(i))
+
+    @staticmethod
+    def export_flatten_with_default_axis():
+        node = onnx.helper.make_node(
+            'Flatten',
+            inputs=['a'],
+            outputs=['b'],
+        )
+        
+        shape = (5, 4, 3, 2)
+        axis_default_value = 1
```

Comment:
This code is going to appear in the doc, I think it would be more clear to explicitly write down the `new_shape`.

Reply:
I agree. Updated as suggested.

Reply:
@onnxbot I thought it's a real bot, but looks like it's not? ahahahaha.

Reply:
@linkerzhang lol it was me sorry. Reposing my comment:

Nice. I think the line `axis_default_value = 1` becomes redundant now.